packages/office-ui-fabric-react/src/components/Tooltip/Tooltip.base.tsx

```diff
@@ -68,6 +79,20 @@ export class TooltipBase extends BaseComponent<ITooltipProps, any> {
     );
   }
 
+  public componentDidMount(): void {
+    TooltipBase._currentVisibleTooltip = this;
+  }
+
+  public componentWillUnmount(): void {
+    TooltipBase._currentVisibleTooltip = undefined;
```

Comment:
I think this should be careful to only clear it out if `_currentVisibleTooltip === this`. Otherwise something else could have rendered and timing issues and such, and now this variable points to another instances resulting in undefined behaviors.